data/scripts/talkactions/gm/teleport_to_player.lua

```diff
@@ -0,0 +1,61 @@
+local teleportToCreature = TalkAction("/listplayers")
```

Comment:
`listplayers` doesn't sound like a command that would teleport me. I almost want this to be part of `/goto` and make that command smarter, but maybe that's too much extra work for not a lot of benefit.